app/src/org/commcare/utils/SigningUtil.java

```diff
@@ -116,6 +116,20 @@ private static void assertWhitelistedUrlHost(String urlString) {
      */
     public static String verifyMessageAndBytes(String message, byte[] signature) throws Exception {
         String keyString = GlobalConstants.TRUSTED_SOURCE_PUBLIC_KEY;
+        return verifyMessageAndBytes(keyString, message, signature);
+    }
+
+
+    /**
+     * Given the link and signature, verify the link using the public key
+     *
+     * @param message   the download link
```

Comment:
this javadoc comment seems wrong to me; `message` isn't a download link, it's just a string